input/pagecontent/changes.md

```diff
@@ -39,17 +37,32 @@ To help implementers, only the more significant changes are listed here.
 <ul>
 <li><a href="SearchParameter-indigenous-status.html">AustralianIndigenousStatus</a></li>
 <li><a href="SearchParameter-encounter-discharge-disposition.html">EncounterDischargeDisposition</a></li>
-<li><a href="SearchParameter-patient-gender-identity.html">PatientGenderIdentity</a></li>
+<li><a href="SearchParameter-gender-identity.html">GenderIdentity</a> (<a href="https://jira.hl7.org/browse/FHIR-43718">FHIR-43718</a>)</li>
 </ul>
 </li>
 <li>Changed Patient.extension in <a href="StructureDefinition-au-patient.html">AU Base Patient</a>: 
-<ul><li>to add Individual Pronouns and Person Recorded Sex Or Gender</li>
+<ul><li>to add Individual Pronouns (<a href="https://jira.hl7.org/browse/FHIR-43719">FHIR-43719</a>) and Person Recorded Sex Or Gender</li>
 <li>to add Australian Veteran Status</li>
-<li>to remove the value slice with the binding to Gender Identity Response (extensible) from genderIdentity (<a href="https://github.com/hl7au/au-fhir-base/issues/815">au-fhir-base #815</a>)</li>
+<li>to remove genderIdentity extension and replace it with the Individual Gender Identity extension (<a href="https://jira.hl7.org/browse/FHIR-43718">FHIR-43718</a>)</li>
 </ul>
 <li>Changed context of extension <a href="StructureDefinition-indigenous-status.html">Australian Indigenous Status</a> to add Person and RelatedPerson.</li>
```

Comment:
Restructure the change log to be a list by profile for the profile changes to make this more readable,

Reply:
This has not been addressed. Should be:
- Change to AU Base Patient:
   - change
   - change
- Changes to AU Base Practitioner:
   - change
   - change that is a group of changes:
      - change
      - change

Reply:
Done

Reply:
Might not be pushed. The file in this PR does not have this content.

Reply:
I'll double check

Reply:
We can merge and then fix after.